libxml2.sh

```
@@ -1,26 +1,30 @@
 package: libxml2
 version: "%(tag_basename)s"
-tag: v2.9.3
+tag: v2.10.4
 build_requires:
-  - "autotools:(slc6|slc7)"
+  - CMake
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)